.eslintrc

```diff
@@ -83,12 +83,13 @@
     "react/prefer-stateless-function": "off",
     "react/jsx-indent-props": "off",
     "react/jsx-closing-bracket-location": "off",
+    // FIXME: Switch using .jsx for jsx files https://github.com/mozilla/addons-frontend/issues/969.
```

Comment:
It looks like this was copied from the old code and I don't think it's relevant. Can it be removed?